docs/user/site.rst

```diff
@@ -172,7 +173,8 @@ next_node \: package
     omitted, there will be no IPv4 or IPv6 anycast address. The MAC address
     defaults to ``16:41:95:40:f7:dc``; this value usually doesn't need to be
     changed, but it can be adjusted to match existing deployments that use a
-    different value.
+    different value. Each entry in the ``name``-field will be resolved as the
+    ipv4 and ipv6-address.
```

Comment:
* "will be resolved to the IPv4 and the IPv6 address"
* This needs more explanation (that this only does something when the clients use the node as resolver)

Reply:
done.